src/sentry/static/sentry/app/views/stream/actions.jsx

```diff
@@ -58,12 +59,21 @@ const getConfirm = (numIssues, allInQuerySelected, query, queryCount) => {
         <p style={{marginBottom: '20px'}}>
           <strong>{question}</strong>
         </p>
-        <ExtraDescription
-          all={allInQuerySelected}
-          query={query}
-          queryCount={queryCount}
```

Comment:
Was this removed b/c of a merge? I'm guessing you want to still pass it `queryCount`